docs/diagrams/ReportArchitectureAuthorship.puml

```diff
@@ -0,0 +1,46 @@
+@startuml
+hide footbox
+skinparam monochrome true
+skinparam Shadowing false
+
+Participant ":Summary\n(v_summary.js)" as summary
+Participant ":Main\n(window.app)" as main
+Participant ":Authorship\n(v_authorship.js)" as authorship
+Participant ":API\n(api.js)" as api
+Participant ":Segment\n(v_segment.js)" as segment
+
+-> summary : View code icon clicked
```

Comment:
Just wondering why only this line starts with capitalized character (inherited from the original DG), otherwise LGTM

Reply:
Thanks, resolved.